sources/qa-lang-main.php

```diff
@@ -145,10 +145,10 @@
        'selected'=>'izbrano', 
        'send_button'=>'Pošlji', 
        'since_x'=>'od ^', //since ^
-       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
-       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
-       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
-       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
+       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
+       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
+       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
+       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
```

Comment:
Moralo bi biti to to. Zadnje spremembe sem naredil v patch 4 , če vam to
kaj pomaga.

2015-04-13 21:28 GMT+02:00 Matjaž M. <notifications@github.com>:

> In sources/qa-lang-main.php
> <https://github.com/matmr/q2a-prevajanje-FS/pull/9#discussion_r28270551>:
>
> > @@ -145,10 +145,10 @@
> >         'selected'=>'izbrano',
> >         'send_button'=>'Pošlji',
> >         'since_x'=>'od ^', //since ^
> > -       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
> > -       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
> > -       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
> > -       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
> > +       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
> > +       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
> > +       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
> > +       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
>
> No super. Smo rešili! Dajte še enkrat pozorno prebrat pa mi sporočite ko
> bo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matmr/q2a-prevajanje-FS/pull/9/files#r28270551>.
>